tests/spec/fs.copyFile.spec.js

```diff
@@ -0,0 +1,52 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.copyFile', function(){
+  beforeEach(function(done){
+    util.setup(function() {
+      var fs = util.fs();
+      fs.writeFile('/srcfile', 'This is a src file.', function(error){
+        if(error) throw error;
+        fs.writeFile('/destfile', 'This is a dest file.', function(error){
+          if(error) throw error;
+          done();
+        });
+      });
+    });
+  });
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.copyFile).to.be.a('function');
+  });
+
+  it('should return an error if the src path does not exist', function(done){
+    var fs = util.fs();
+    var src = null;
```

Comment:
This is prone to being incorrect changed in the future, since it looks like you just forgot to initialize your variable.  You need to document that you meant to do this.  A few options:

1. `var src; // intentionally left undefined`
1. `fs.copyFile(undefined /* intentionally undefined */, '/dest.txt', function(error) {`